dataframe-arrow/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/ArrowKtTest.kt

```diff
@@ -558,23 +565,26 @@ internal class ArrowKtTest {
         }
     }
 
-    @Test
-    fun testArrowReaderExtension() {
+    private fun expectedSimpleDataFrame(): AnyFrame{
```

Comment:
missed a space before "{"

Reply:
Fixed, weird thing that was not detected by linter during my local build

Reply:
Yes, our linter is a bit broken :/ https://github.com/Kotlin/dataframe/issues/364